arangod/Agency/Node.h

```diff
@@ -75,6 +75,60 @@ typedef std::chrono::steady_clock::time_point SteadyTimePoint;
 
 class Store;
 
+class SmallBuffer {
+  uint8_t* _start;
+  size_t _size;
+ public:
+  SmallBuffer() : _start(nullptr), _size(0) {}
+  SmallBuffer(size_t size) : _start(new uint8_t[size]), _size(size) {}
+  SmallBuffer(SmallBuffer const& other) {
+    if (other.empty()) {
+      _start = nullptr;
+      _size = 0;
+    } else {
+      _start = new uint8_t[other._size];
+      _size = other._size;
+      memcpy(_start, other._start, other._size);
+    }
+  }
```

Comment:
nitpicking... it was already fine as it was before, but we could make sure we always initialize the instance variables in the initializer list. but it is fine as it was before.
```suggestion
  SmallBuffer(SmallBuffer const& other) 
      : SmallBuffer() {
    if (!other.empty()) {
      _start = new uint8_t[other._size];
      _size = other._size;
      memcpy(_start, other._start, other._size);
    }
  }
```